spreadsheet/context.go

```diff
@@ -233,6 +233,7 @@ func (e *evalContext) LastRow(col string) int {
 var refRegexp *regexp.Regexp = regexp.MustCompile(`^([a-z]+)([0-9]+)$`)
 
 func validateRef(cr string) bool {
+	cr = strings.ReplaceAll(cr, "$", "")
```

Comment:
Please check build errors:
# github.com/unidoc/unioffice/spreadsheet
../../../spreadsheet/context.go:236:7: undefined: strings.ReplaceAll

I think ReplaceAll is new, better use Replace with -1.

Reply:
fixed